src/test/java/test_cases/TestCase02.java

```diff
@@ -0,0 +1,27 @@
+package test_cases;
+
+import common.helpers.Constant;
+import model.User;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.GeneralPage;
+import page_objects.LoginPage;
+
+public class TestCase02 extends BaseTest {
+  GeneralPage generalPage = new GeneralPage();
+  LoginPage loginPage = new LoginPage();
+
+  @Test(description = "User can't login with blank Username textbox")
+  public void test() {
```

Comment:
what does `test()`?

Reply:
I removed class TestCase2 and I merge 2 class into LoginTests